CMakeLists.txt

```diff
@@ -127,6 +127,7 @@ option(SUPPORT_PCX "Support loading PCX images" ON)
 option(SUPPORT_PNG "Support loading PNG images" ON)
 option(SUPPORT_PNM "Support loading PNM images" ON)
 option(SUPPORT_QOI "Support loading QOI images" ON)
+option(SUPPORT_STB "Support loading STB images" ON)
```

Comment:
this isn't enabled by autotools, it's not even in autotools and other build systems,
so it should default to disabled at the least.

Reply:
Changed.
`IMG_stb.c` is also a bit inconsistent.
All other `IMG_xyz.c` files need a `LOAD_xyz` definition, where `stb` needs `USE_STBIMAGE`.

Reply:
AFAICS, it's a backend selector, not a format enabler